samples/hybrid-connections/java/role-based-access-control/pom.xml

```diff
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure.microsoft</groupId>
+  <artifactId>relay-rbac-sample</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>Azure Relay Role Based Access Control Sample</name>
+  
+  <dependencies>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-relay</artifactId>
+      <version>0.0.3</version>
```

Comment:
The version of the Java client should be bumped to 0.0.4 for the addition of RBAC changes. Putting this for now and will need to be changed once the package is officially released.